### PR TITLE
[Specs] Spec cleanup

### PR DESCRIPTION
**Pull request by eliasjpr**
- Breaks controller specs into smaller parts
- Removes duplicate settings spec
- Move helper method to Controller helpers

### CODE REVIEW

File: spec/amber/environment/env_spec.cr [hunk 1]
```diff
@@ -24,7 +24,7 @@ module Amber::Environment
         amber_env.{{env.id}}.should_not be_nil
       end
     end
-  {% end %}
+    {% end %}
```
Comment (marksiemers) [inline spec/amber/environment/env_spec.cr]: Is this meant to be moved over?

File: spec/amber/controller/respond_width_spec.cr [hunk 2]
```diff
@@ -0,0 +1,107 @@
+require "../../../spec_helper"
```
Comment (marksiemers) [inline spec/amber/controller/respond_width_spec.cr]: Typo in the filename `spec/amber/controller/respond_width_spec.cr`

File: spec/amber/environment/env_spec.cr (hunk 1 above)
Comment (eliasjpr) [reply spec/amber/environment/env_spec.cr]: Yes, to align the with the opening if. 

I just noticed that this file is named incorrectly

**Comment by eliasjpr** [comment]
@marksiemens they are modules already and those unit/integration specs. Does it make sense? I can improve the specs

**Pull request merged**